optima/model.py

```diff
@@ -782,8 +781,8 @@ def model(simpars=None, settings=None, verbose=None, benchmark=False, die=True):
                     change[tx[cd4],:]  = dT[cd4]
             people[:,:,t+1] = people[:,:,t] + change # Update people array
             newpeople = popsize[:,t+1]-people[:,:,t+1].sum(axis=0) # Number of people to add according to simpars['popsize'] (can be negative)
-            people[susreg,pop,t+1] += newpeople[pop] - numcirc[pop,t] # Add new people, then subtract if from circumcision
-            people[circ,pop,t+1]   += numcirc[pop,t] # And add these people into the circumcised compartment
+            people[susreg,:,t+1] += newpeople - numcirc[pop,t] # Add new people, then subtract if from circumcision
```

Comment:
it's not in a population loop -- it used to be, but it's not any mo -- AUGH, still a bug